composables/helpers/buildOrdersFilterInput.ts

```diff
@@ -1,3 +1,4 @@
+import { ref } from "@vue/composition-api"
 export const buildOrdersFilterInput = (params: {
```

Comment:
we need unit tests around this  buildOrdersFilterInput  function

Reply:
Resolved